src/rev_ai/apiclient.py

```diff
@@ -324,6 +325,18 @@ def get_account(self):
         """
         url = urljoin(self.base_url, 'account')
         response = self.session.get(url)
-        response.raise_for_status()
+        self._api_failure_handler(response)
 
         return Account.from_json(response.json())
+
+    def _api_failure_handler(self, response):
```

Comment:
This is named as if it's a class. Functions should be named as actions
ie `_handle_api_failure`